### PR TITLE
New approach to create docker for ARM: Don't compile in arm

### PR DESCRIPTION
**Pull request by mappuji**
New approach to create docker for ARM: Don't compile in arm

***
Approach:
* Extracted compiled artifact of planet from what we created in `planet-test` and add it to `rpi-Dockerfile`

Fixes to #1417 

***
Working examples:
* Live site: http://kraken.ole.org/eng/
* Build: https://travis-ci.org/open-learning-exchange/planet/jobs/395724940 (shrink 6 minutes to only 2 minutes)

***
The proposed commit to be committed is only until this commit https://github.com/open-learning-exchange/planet/pull/1444/commits/ac08af49bae3b3e43c894783466d5f0e335878cb

This commit https://github.com/open-learning-exchange/planet/pull/1444/commits/6b41e56fea8b7d01150ca243660ce0d0db33eee1 only for testing and demonstrating.

***
Related PR #1435 

***

Try this to test `docker-compose.yml`

```yaml
services:
  couchdb:
    expose:
      - 5984
    image: treehouses/couchdb:2.1.1
    ports:
      - 2200:5984
  db-init:
    image: treehouses/planet-tags:rpi-db-init-0.2.1-localization-docker-efd94cd3
    depends_on:
      - couchdb
    environment:
      - COUCHDB_HOST=http://couchdb:5984
  planet:
    image: treehouses/planet-tags:rpi-0.2.3-dont_compile_in_arm-95b0924b
    ports:
      - 80:80
    environment:
      - HOST_PROTOCOL=http
      - DB_HOST=kraken.ole.org
      - DB_PORT=2200
      - CENTER_ADDRESS=earth.ole.org:2200
    #volumes:
    #  - ./planet/default.conf:/etc/nginx/conf.d/default.conf
    depends_on:
      - couchdb
version: "2"
```

### CODE REVIEW

File: .travis/travis_utils.sh [hunk 1]
```diff
@@ -108,7 +108,7 @@ package_docker(){
   # $2: tag
   # $3: tag latest
   build_message processing $2
-  if [ "$TRAVIS_BRANCH" = "master" ] || [ ! -z "$gtag" ] || [ ! -z "$TRAVIS_TAG" ]; then
+  if [ "$TRAVIS_BRANCH" = "dont_compile_in_arm" ] || [ ! -z "$gtag" ] || [ ! -z "$TRAVIS_TAG" ]; then
```
Comment (lmmrssa) [inline .travis/travis_utils.sh]: @mappuji @sahilph @dogi @paulbert I think we agreed to have multi language build only for branches with specific tag probably release or other tags.
Right now it is always building for specific branch or other branches if it has tag.

Comment (mappuji) [reply .travis/travis_utils.sh]: I'll define one

**Comment by mappuji** [comment]
Sure, I'll define one @dogi @lmmrssa

File: .travis/travis_utils.sh (hunk 1 above)
Comment (sahilph) [reply .travis/travis_utils.sh]: Yes, a new tag for multi language would be better.

**Comment by mappuji** [comment]
@lmmrssa  `$TRAVIS_TAG` is default environment variable from Travis if there is a tag. The `$gtag` is actually `$TRAVIS_TAG` that we passed to rpi build script.

**Comment by lmmrssa** [comment]
@mappuji does that mean we are not building language only when we have tag on branch? Right now it seems like it is always building multi language.

**Comment by sahilph** [comment]
@lmmrssa It will only build multi language when we create a new version (via tag).

**Comment by mappuji** [comment]
you can try @lmmrssa if not sure

**Pull request merged**